osu.Game/Database/ArchiveModelManager.cs

```diff
@@ -110,7 +111,7 @@ public Task Import(params string[] paths)
         protected async Task Import(ProgressNotification notification, params string[] paths)
         {
             notification.Progress = 0;
-            notification.Text = "Import is initialising...";
+            notification.Text = $"{HumanisedModelName.Humanize().Pluralize(paths.Length == 1)} import is initialising...";
```

Comment:
This should not be plural. Also you probably want `.Humanize(LetterCasing.Title)`.